hopo/semesterticket.tex

```diff
@@ -1,26 +1,87 @@
+Seit Herbst 2008 verhandeln das Verkehrsreferat der Studierendenvertretung und
+das Studierendenwerk mit dem VRN über einen neuen Vertrag. Die Studis fordern ein
+Ende der enormen Preissteigerungen, um auch in Zukunft mit dem Semesterticket
+ein günstiges Nahverkehrsangebot zu gewährleisten. Da der VRN jedoch weiter an
+Preissteigerungen von ca. 10\% pro Jahr festhalten will, stand das
+Semesterticket vor dem Aus. 
+Nach einem Übergangsvertrag, auf den der VRN sich im WS 09/10 in letzter Minute
+eingelassen hat, wurde im WS 13/14 erneut verhandelt. Diesmal kam der
+Kommunalwahlkampf in Heidelberg den Studis zu Gute, aber trotz massiven Drucks
+aus der Politik und Zuschüssen durch den Gemeinderat ist das Semesterticket
+nach wie vor recht teuer und der VRN wird die Preise weiterhin regelmäßig
+erhöhen.
+
+Eine Neuerung bei den letzten Vertragsverhandlungen war, dass der frisch
+eingerichtete \gls{StuRa} alle Studis zu einer Urabstimmung über das
+Vertragsangebot aufgerufen hat. Auch wenn die Urabstimmung wegen teils schwerer
+Formfehler von verschiedenen Seiten angefochten wurde, hat sich doch eine
+Mehrheit der Abstimmenden für das Semesterticket ausgesprochen, sodass der
+StuRa dem Vertragsangebot letztendlich zugestimmt hat.
+
+Mittlerweile steigt das Semesterticket im Preis jedes Semester. Der StuRa und
+vor allem dessen Verkehrsreferat setzt sich dafür ein, die Preise so gering wie
+möglich zu halten, was kein leichtes Unterfangen ist.\\[1em]
 
-Die \gls{FSK} und das Studentenwerk versuchen für die Studierenden ein bestmögliches Ticket zu erreichen. Dies bedeutet nicht, dass diese einem inakzeptablen Vertrag zustimmen werden, an den die Studierenden die nächsten fünf Jahre gebunden sind. Selbstverständlich wird dabei aber der Tatsache Rechnung getragen, dass ein Großteil der Studierenden auf den ÖPNV angewiesen ist.\\[2mm]
+Heidelberg ist eine eher kleine übersichtliche Stadt in welcher alles bequem
+mit dem Fahrrad erledigt werden können -- meist sogar schneller als mit Bus und
+Bahn. Dank der milden Temperaturen ist dies auch im Winter durchaus möglich.
+Wenn ihr in Heidelberg selbst wohnt ist daher gut zu überlegen ob sich ein
+Semesterticket lohnt oder man wie viele Andere das Fahrrad nutzt.
 
-Heidelberg ist eine eher kleine übersichtliche Stadt in welcher alle Weg bequem mit dem Fahrrad erledigt werden können -- meist sogar schneller als mit Bus und Bahn. Dank der milden Temperaturen ist dies auch im Winter durchaus möglich. Wenn ihr in Heidelberg selbst wohnt ist daher gut zu überlegen ob sich ein Semesterticket lohnt oder man wie viele Andere das Fahrrad nutzt.
+Ebenfalls wurde im Sommersemester 2016 eine Umfrage zu einem Sockelbeitrag von
+\EUR{2,40} für die Nutzung von NextBike (in ganz Deutschland, überall dort wo
+es NextBike gibt) gleichzeitig zu den Wahlen durchgeführt. Dies wurde sehr
+knapp (52\% zu 48\%) abgelehnt. Natürlich ist dies nicht aus der Welt und es
+wird weiter im Sinne der Studierenden zu diesem Thema verhandelt.
```

Comment:
klingt etwas pro NextBike. Da es durchaus kontroversen in der Fachschaft zu diesem Thema gab sollte das Ersti-Info da sehr neutral sein.